content/microcontrollers/pybadge.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Adafruit PyBadge"
+weight: 3
+---
+
+The [Adafruit PyBadge](https://www.adafruit.com/product/4200) is a ARM development board based on the Atmel [ATSAMD51J19A](https://www.microchip.com/wwwproducts/en/ATSAMD51J19A) family of SoC.
+
+It has many built-in devices, such as a 1.8" 160x128 Color TFT Display, 8 x buttons, 5 x NeoPixels, a triple-axis accelerometer, a light sensor, and a speaker.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | YES | YES |
+| I2C      | YES | YES |
+| ADC      | YES | YES |
+| PWM      | YES | YES |
+
+## Machine Package Docs
+
+[Documentation for the machine package for the Adafruit PyBadge](../machine/pybadge)
+
+## Flashing
+
+### UF2
+
+The PyBadge comes with the [UF2 bootloader](https://github.com/Microsoft/uf2) already installed.
+
+### CLI Flashing on Linux
+
+- Plug your PyBadge into your computer's USB port.
+- Flash your TinyGo program to the board using this command:
+
+    ```shell
+    tinygo flash -target=pybadge [PATH TO YOUR PROGRAM]
+    ```
+
+- The PyBadge board should restart and then begin running your program.
+
+### CLI Flashing on macOS
+
+- Plug your PyBadge into your computer's USB port.
+- Flash your TinyGo program to the board using this command:
+
+    ```shell
+    tinygo flash -target=pybadge [PATH TO YOUR PROGRAM]
+    ```
+
+- The PyBadge board should restart and then begin running your program.
+
+### CLI Flashing on Windows
+
+- Plug your PyBadge into your computer's USB port.
+- Double tap the "RESET" button on the board.
+- Wait until the PyBadge board appears as a flash drive.
+- Flash your TinyGo program to the board using this command:
+
+    ```shell
+    tinygo flash -target=pybadge [PATH TO YOUR PROGRAM]
```

Comment:
I guess this depends on https://github.com/tinygo-org/tinygo/pull/786?

Reply:
That is correct.